core/drivers/clk/clk-stm32mp13.c

```diff
@@ -0,0 +1,1727 @@
+		      RCC_PLLNCR_DIVREN | RCC_PLLNCR_PLLON;
+
+	vco->frac = _fdt_read_uint32_default(fdt, subnode, "frac", 0);
+
+	vco->src = _fdt_read_uint32_default(fdt, subnode, "src", UINT32_MAX);
+
+	return 0;
+}
+
+static int clk_stm32_load_output_config(const void *fdt, int subnode,
+					struct stm32_pll_output *output)
+{
+	return _fdt_read_uint32_array(fdt, subnode, "st,pll_div_pqr",
+				      output->output, (int)PLL_DIV_PQR_NB);
+}
+
+static int clk_stm32_parse_pll_fdt(const void *fdt, int subnode,
+				   struct stm32_pll_dt_cfg *pll)
+{
+	const fdt32_t *cuint = NULL;
+	int subnode_pll = 0;
+	int subnode_vco = 0;
+	int err = 0;
+
+	cuint = fdt_getprop(fdt, subnode, "st,pll", NULL);
+	if (!cuint)
+		return -FDT_ERR_NOTFOUND;
+
+	subnode_pll = fdt_node_offset_by_phandle(fdt, fdt32_to_cpu(*cuint));
+	if (subnode_pll < 0)
+		return -FDT_ERR_NOTFOUND;
+
+	cuint = fdt_getprop(fdt, subnode_pll, "st,pll_vco", NULL);
+	if (!cuint)
+		return -FDT_ERR_NOTFOUND;
+
+	subnode_vco = fdt_node_offset_by_phandle(fdt, fdt32_to_cpu(*cuint));
+	if (subnode_vco < 0)
+		return -FDT_ERR_NOTFOUND;
+
+	err = clk_stm32_load_vco_config(fdt, subnode_vco, &pll->vco);
+	if (err != 0)
+		return err;
+
+	err = clk_stm32_load_output_config(fdt, subnode_pll, &pll->output);
+	if (err != 0)
+		return err;
+
+	return 0;
+}
+
+static int stm32_clk_parse_fdt_all_pll(const void *fdt, int node,
+				       struct stm32_clk_platdata *pdata)
+{
+	size_t i = 0;
+
+	for (i = PLL1_ID; i < pdata->npll; i++) {
+		struct stm32_pll_dt_cfg *pll = pdata->pll + i;
+		char name[RCC_PLL_NAME_SIZE];
+		int subnode = 0;
+		int err = 0;
+
+		snprintf(name, sizeof(name), "st,pll@%d", i);
+
+		subnode = fdt_subnode_offset(fdt, node, name);
+		if (!_fdt_check_node(fdt, subnode))
+			continue;
+
+		err = clk_stm32_parse_pll_fdt(fdt, subnode, pll);
+		if (err != 0)
+			panic();
+	}
+
+	return 0;
+}
+
+static int stm32_clk_parse_fdt_opp(const void *fdt, int node,
+				   const char *opp_name,
+				   struct stm32_clk_opp_cfg *opp_cfg)
+{
+	int subnode = 0;
+	int nb_opp = 0;
+
+	node = fdt_subnode_offset(fdt, node, opp_name);
+	if (!_fdt_check_node(fdt, node))
+		return 0;
+
+	fdt_for_each_subnode(subnode, fdt, node) {
+		if (nb_opp >= MAX_OPP) {
+			EMSG("%d MAX opp in %s !", MAX_OPP, opp_name);
+			break;
+		}
+
+		opp_cfg->frq = _fdt_read_uint32_default(fdt, subnode,
+							"hz",
+							UINT32_MAX);
+
+		opp_cfg->src = _fdt_read_uint32_default(fdt, subnode,
+							"st,clksrc",
+							UINT32_MAX);
+
+		opp_cfg->div = _fdt_read_uint32_default(fdt, subnode,
+							"st,clkdiv",
+							UINT32_MAX);
+
+		clk_stm32_parse_pll_fdt(fdt, subnode, &opp_cfg->pll_cfg);
+
+		opp_cfg++;
+		nb_opp++;
+	}
+
+	return 0;
+}
+
+static int stm32_clk_parse_fdt_all_opp(const void *fdt, int node,
+				       struct stm32_clk_platdata *pdata)
+{
+	struct stm32_clk_opp_dt_cfg *opp = pdata->opp;
+
+	node = fdt_subnode_offset(fdt, node, "st,clk_opp");
+	if (!_fdt_check_node(fdt, node))
```

Comment:
add braces when instruction block is more than 1 more including inline comment. 

with the comment on `_fdt_check_node()`:
```
	node = fdt_subnode_offset(fdt, node, "st,clk_opp");
	if (node < 0) {
		/* No opp are defined */
		return 0;
	}